### ISSUE TITLE
the requirements in python 3.7

### ISSUE THREAD

**Issue by Dittonal** [NREL/alfabet#17]
absl-py==0.15.0
alfabet==0.4.1
astunparse==1.6.3
backcall==0.2.0
cached-property==1.5.2
cachetools==5.3.0
certifi==2022.12.7
charset-normalizer==3.1.0
clang==5.0
colorama==0.4.6
debugpy==1.6.6
decorator==5.1.1
entrypoints==0.4
flatbuffers==1.12
gast==0.4.0
google-auth==2.16.2
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.51.3
h5py==3.1.0
idna==3.4
importlib-metadata==6.1.0
ipykernel==6.16.2
ipython==7.34.0
jedi==0.18.2
joblib==1.2.0
jupyter_client==7.4.9
jupyter_core==4.12.0
keras==2.6.0
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.2
libclang==15.0.6.1
Markdown==3.4.1
MarkupSafe==2.1.2
matplotlib-inline==0.1.6
nest-asyncio==1.5.6
networkx==2.6.3
nfp==0.3.12
numpy==1.19.5
oauthlib==3.2.2
opt-einsum==3.3.0
packaging==23.0
pandas==1.3.5
parso==0.8.3
pickleshare==0.7.5
Pillow==9.4.0
platformdirs==3.1.1
pooch==1.7.0
prompt-toolkit==3.0.38
protobuf==3.19.6
psutil==5.9.4
pyasn1==0.4.8
pyasn1-modules==0.2.8
Pygments==2.14.0
python-dateutil==2.8.2
pytz==2022.7.1
pywin32==305
PyYAML==6.0
pyzmq==25.0.1
rdkit==2022.9.5
requests==2.28.2
requests-oauthlib==1.3.1
rsa==4.9
scikit-learn==0.24.2
scipy==1.7.3
six==1.15.0
tensorboard==2.11.2
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.6.0
tensorflow-estimator==2.11.0
tensorflow-io-gcs-filesystem==0.31.0
termcolor==1.1.0
tf-estimator-nightly==2.8.0.dev2021122109
threadpoolctl==3.1.0
tornado==6.2
tqdm==4.65.0
traitlets==5.9.0
typing-extensions==3.7.4.3
urllib3==1.26.15
wcwidth==0.2.6
Werkzeug==2.2.3
wincertstore==0.2
wrapt==1.12.1
zipp==3.15.0
